### PR TITLE
Add Galar pokemon data

### PR DESCRIPTION
**Pull request by RedSparr0w**
Evolutions are the only thing not added currently

### CODE REVIEW

File: src/scripts/pokemons/PokemonList.ts [hunk 1]
```diff
@@ -14182,6 +14182,1586 @@ const pokemonList: PokemonListData[] =
...
+                PokemonType.Steel,
+            ],
+            'base': {
+                'hitpoints': 92,
+                'attack': 170,
+                'specialAttack': 80,
+                'defense': 115,
+                'specialDefense': 115,
+                'speed': 148,
+            },
```
Comment (anapple96) [inline src/scripts/pokemons/PokemonList.ts:15658]: We want special forms?

File: src/scripts/pokemons/PokemonList.ts [hunk 2]
```diff
@@ -14182,6 +14182,1586 @@ const pokemonList: PokemonListData[] =
...
+            'eggCycles': 120,
+            'levelType': LevelType.slow,
+            'exp': 335,
+            'catchRate': 10,
+        },
+        {
+            'id': 889,
+            'name': 'Zamazenta\nCrowned Shield',
+            'type': [
```
Comment (apple096) [inline src/scripts/pokemons/PokemonList.ts:15667]: May want to add the special stuffs later, in a separate PR

Comment (RedSparr0w) [reply src/scripts/pokemons/PokemonList.ts:15667]: Nah, not supposed to be there.

Comment (apple096) [reply src/scripts/pokemons/PokemonList.ts:15667]: Zamazenta/Zacian's normal form = Heroes of Many Battles, may want to add them in, as they're not in yet

File: src/scripts/pokemons/PokemonList.ts [hunk 3]
```diff
@@ -14182,6 +14182,1586 @@ const pokemonList: PokemonListData[] =
...
+            'id': 888,
+            'name': 'Zacian',
+            'type': [
+                PokemonType.Fairy,
+                PokemonType.Steel,
+            ],
+            'base': {
+                'hitpoints': 92,
+                'attack': 170,
+                'specialAttack': 80,
```
Comment (apple096) [inline src/scripts/pokemons/PokemonList.ts:15654]: May want to fix the stats as well, as Crowned Forms/Other forms have different stats

Comment (RedSparr0w) [reply src/scripts/pokemons/PokemonList.ts:15654]: Yeah, wasn't finished with remove forms

**Comment by RedSparr0w** [comment]
Sweet, should be good now.

**Review by apple096 — APPROVED** [review]

**Pull request merged**